resources/charge.js

```diff
@@ -100,4 +100,16 @@ Charge.prototype.removeDiscount = function removeDiscount(charge_id, params){
   return this.recharge.request(url, 'POST', this.key, params);
 }
 
+/**
+ * Process the charge using ChargeID.
+ *
+ * @param {Number} id Charge ID
+ * @return {Promise} Promise that resolves with the result
+ * @public
+ */
+Charge.prototype.removeDiscount = function removeDiscount(charge_id){
```

Comment:
Is the function name correct here? Seems like `removeDiscount` is already present. Looks like the function name should be processCharge according to the description